selfdrive/test/docker_tag_multiarch.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -lt 2 ]; then
+  echo "Usage: $0 <base|docs|sim|prebuilt|cl> <arch1> <arch2> ..."
+  exit 1
+fi
+
+if [ $1 = "base" ]; then
```

Comment:
can you make a common script for these checks and source it in both the scripts?